lib/suspenders/version.rb

```diff
@@ -1,5 +1,5 @@
 module Suspenders
-  RAILS_VERSION = "~> 4.2.0"
+  RAILS_VERSION = '~> 4.2.0'
   RUBY_VERSION = IO.read("#{File.dirname(__FILE__)}/../../.ruby-version").strip
-  VERSION = "1.35.0"
+  VERSION = '1.35.3'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.